src/transformState.js

```diff
@@ -45,8 +45,25 @@
  * @param {Object} state
  * @param {Object[]} transforms
  */
+
 function transformState(state, transforms) {
-  // write code here
+  for (let i = 0; i < transforms.length; i++) {
+    if (transforms[i].operation === 'addProperties') {
```

Comment:
строки прийнято виносити в окрему змінну константу, для подальшого використання в кількох місцях.
 ```
const OPERATIONS = {
  add: 'addProperties',
  remove: 'removeProperties',
  clear: 'clear'
}
```
потім ти можеш використовувати це ось так 
```suggestion
    if (transforms[i].operation === OPERATIONS.add) {
```
І якщо ці операції будуть використовувати в 2-3 місцях в програмі і в разі якщо треба буде змінити строку на щось інше, тобі не треба буде шукати і робити зміни по всіх файлах, достатньо змінити тільки строку в константі.